recipes-debian/postfix/postfix_debian.bb

```diff
@@ -0,0 +1,309 @@
+
+    install -d ${D}${sysconfdir}/ufw/applications.d
+    install -m 0755 ${S}/debian/postfix.ufw.profile ${D}${sysconfdir}/ufw/applications.d/postfix
+
+    install -d ${D}${systemd_unitdir}/system-generators
+    install -m 0755 ${S}/debian/postfix-instance-generator ${D}${systemd_unitdir}/system-generators
+
+    install -d ${D}${systemd_system_unitdir}
+    install -m 0644 ${S}/debian/postfix@.service ${D}${systemd_system_unitdir}
+
+    install -m 0755 ${S}/bin/rmail ${D}${bindir}
+
+    install -d ${D}${libdir}/postfix
+
+    install -m 0755 ${S}/debian/configure-instance.sh ${D}${libdir}/postfix
+
+    install -m 0755 ${S}/lib/libmilter.a ${D}${libdir}/postfix
+    install -m 0755 ${S}/lib/libxsasl.a ${D}${libdir}/postfix
+
+    install -m 0755 ${S}/debian/postfix_groups.pl ${D}${libdir}/postfix
+
+    cp -r --no-preserve=ownership ${D}${libexecdir}/postfix ${D}${libdir}/postfix/sbin
+    install -m 0755 ${S}/libexec/fsstone ${D}${libdir}/postfix/sbin
+
+    install -m 0755 ${S}/bin/sendmail ${D}${libdir}/sendmail
+
+    install -m 0755 ${S}/bin/rmail ${D}${sbindir}
+
+    install -m 0755 ${S}/debian/postfix-add-filter ${D}${sbindir}
+    install -m 0755 ${S}/debian/postfix-add-policy ${D}${sbindir}
+    install -m 0755 ${S}/bin/posttls-finger ${D}${sbindir}
+    install -m 0755 ${S}/bin/qmqp-sink ${D}${sbindir}
+    install -m 0755 ${S}/bin/qmqp-source ${D}${sbindir}
+    install -m 0755 ${S}/auxiliary/qshape/qshape.pl ${D}${sbindir}/qshape
+    install -m 0755 ${S}/bin/smtp-source ${D}${sbindir}
+}
+
+do_install_append_class-native() {
+    ln -sf ../sbin/sendmail.postfix ${D}${bindir}/newaliases
+    ln -sf ../sbin/sendmail.postfix ${D}${bindir}/mailq
+}
+
+ALTERNATIVE_${PN}-bin = "sendmail mailq newaliases"
+# /usr/lib/sendmial is required by LSB core test
+ALTERNATIVE_${PN}-bin_linuxstdbase = "sendmail mailq newaliases usr-lib-sendmail"
+ALTERNATIVE_TARGET[mailq] = "${bindir}/mailq"
+ALTERNATIVE_TARGET[newaliases] = "${bindir}/newaliases"
+ALTERNATIVE_TARGET[sendmail] = "${sbindir}/sendmail.postfix"
+ALTERNATIVE_LINK_NAME[sendmail] = "${sbindir}/sendmail"
+ALTERNATIVE_TARGET[usr-lib-sendmail] = "${sbindir}/sendmail.postfix"
+ALTERNATIVE_LINK_NAME[usr-lib-sendmail] = "/usr/lib/sendmail"
+
+ALTERNATIVE_PRIORITY = "120"
+
+ALTERNATIVE_${PN}-doc += "mailq.1 newaliases.1 sendmail.1"
+ALTERNATIVE_LINK_NAME[mailq.1] = "${mandir}/man1/mailq.1"
+ALTERNATIVE_LINK_NAME[newaliases.1] = "${mandir}/man1/newaliases.1"
+ALTERNATIVE_LINK_NAME[sendmail.1] = "${mandir}/man1/sendmail.1"
+
+pkg_postinst_ontarget_${PN}-cfg () {
+    touch /etc/aliases
+    newaliases
+
+    # generate virtual_alias, default is hash
+    touch /etc/postfix/virtual_alias
+    postmap /etc/postfix/virtual_alias
+}
+
+RDEPENDS_${PN}-bin += "python3 python3-modules perl \
+                       perl-module-io-file perl-module-getopt-std \
+                       perl-module-file-find perl-module-file-spec perl-module-file-find \
+                      "
```

Comment:
The qshape command uses the perl modules. (Please refer to "How to test".)
Therefore, I have added the perl modules to RDEPENDS.

Reply:
That's nice. btw, perl-module-file-find is defined twice.

Reply:
I am sorry. I fixed it.